modules/installation-azure-permissions.adoc

```diff
@@ -6,9 +6,15 @@
 [id="installation-azure-permissions_{context}"]
 = Required Azure roles
 
-{product-title} needs a service principal so it can manage Microsoft Azure resources. Before you can create a service principal, your Azure account subscription must have the following roles:
+{product-title} needs a service principal so it can manage Microsoft Azure resources. Before you create a service principal, review the following information:
+
+Your Azure account subscription must have the following roles:
 
 * `User Access Administrator`
 * `Contributor`
 
-To set roles on the Azure portal, see the link:https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal[Manage access to Azure resources using RBAC and the Azure portal] in the Azure documentation.
\ No newline at end of file
+Your Active Directory (AD) must have the following permission:
```

Comment:
How about to change `Your Azure account on Azure Active Directory (AD) must have the following permission:`?